cmd/nginx-meshctl/main.go

```diff
@@ -2,22 +2,176 @@
 package main
 
 import (
+	"bufio"
+	"io"
+	"log"
 	"os"
+	"path/filepath"
+	"regexp"
+	"runtime"
+	"strings"
 
 	"github.com/nginxinc/nginx-service-mesh/internal/nginx-meshctl/commands"
+
+	"github.com/spf13/cobra"
+	"github.com/spf13/cobra/doc"
+	"golang.org/x/text/cases"
+	"golang.org/x/text/language"
 )
 
 var (
 	pkgName = "nginx-meshctl"
 	version = "0.0.0"
 	commit  = "local"
+	genDocs = ""
```

Comment:
Rather than a variable built in, how about a flag that a user specifies, such as `--verbose-usage` or something along those lines?

Reply:
Will do 👍